candidate_login/views.py

```diff
@@ -11,10 +11,10 @@ class CandidateLoginView(auth_views.LoginView):
 
 def login_success(request):
     if request.user.is_authenticated:
-        name = request.user.first_name
-        if name == "" or name is None:
-            name = request.user.username
-        messages.success(request, "Hi, " + name + "!")
+        email = request.user.email
+        if email == "" or email is None:
```

Comment:
Shorthand for string checks can just be 
if not email:
(This is equivalent to -> if email == "" or email is None). 
Also, by convention, do None checks before empty. So it should be:
if email is None or email == "":
But then again, this is much better:
if not email: